<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1009</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-color: #fefaf1;
    color: #5c4033;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>0</y>
      <width>811</width>
      <height>601</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#dashboardPage {
	background-color: #3e2c23;
    color: #f2e4da
}

QWidget#transactionsPage {
	background-color: #3e2c23;
    color: #f2e4da
}

QWidget#budgetsPage {
	background-color: #3e2c23;
    color: #f2e4da
}

QWidget#reportsPage {
	background-color: #3e2c23;
    color: #f2e4da
}</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="dashboardPage">
     <widget class="QLabel" name="incomeLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>236</width>
        <height>101</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Variable Text Semibold</family>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel#incomeLabel {
    background-color: #bfa391;
	border-radius: 10px;
}</string>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="text">
       <string>Nothing</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="indent">
       <number>-1</number>
      </property>
     </widget>
     <widget class="QLabel" name="helloLabel">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>331</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Variable Text Semibold</family>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel#helloLabel {
	color: #bfa391
}</string>
      </property>
      <property name="text">
       <string>Hello, Erfan</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="dateLabel">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>30</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Variable Display</family>
        <pointsize>16</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel#dateLabel {
	color: #bfa391
}</string>
      </property>
      <property name="text">
       <string>April 17, 2025</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="addTransactiondButton">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>30</y>
        <width>181</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Variable Text Semibold</family>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#addTransactiondButton {
background-color: #bfa391;
    color: black;
    border: none;
    border-radius: 10px;
    padding: 8px 12px
}

QPushButton#addTransactiondButton:hover {
    background-color: #a37c68;
}</string>
      </property>
      <property name="text">
       <string>+ Add Transaction</string>
      </property>
     </widget>
     <widget class="QLabel" name="expenseLabel">
      <property name="geometry">
       <rect>
        <x>285</x>
        <y>100</y>
        <width>241</width>
        <height>101</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Variable Text Semibold</family>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel#expenseLabel {
    background-color: #bfa391;
	border-radius: 10px
}</string>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="text">
       <string>Nothing</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="indent">
       <number>-1</number>
      </property>
     </widget>
     <widget class="QLabel" name="balanceLabel">
      <property name="geometry">
       <rect>
        <x>545</x>
        <y>100</y>
        <width>236</width>
        <height>101</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Variable Text Semibold</family>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel#balanceLabel {
    background-color: #bfa391;
	border-radius: 10px
}</string>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="text">
       <string>Nothing</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="indent">
       <number>-1</number>
      </property>
     </widget>
     <widget class="QTableView" name="recentTransactionstableView">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>220</y>
        <width>331</width>
        <height>361</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>220</y>
        <width>411</width>
        <height>361</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="chartLayout"/>
     </widget>
    </widget>
    <widget class="QWidget" name="transactionsPage">
     <widget class="QPushButton" name="addTransactiontButton">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>50</y>
        <width>181</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Variable Text Semibold</family>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#addTransactiontButton {
	background-color: #bfa391;
    color: black;
    border: none;
    border-radius: 10px;
    padding: 8px 12px
}

QPushButton#addTransactiontButton:hover {
    background-color: #a37c68;
}</string>
      </property>
      <property name="text">
       <string>+ Add Transaction</string>
      </property>
     </widget>
     <widget class="QLabel" name="transactionsLabel">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>201</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Variable Text Semibold</family>
        <pointsize>26</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel#transactionsLabel {
	color: #bfa391
}</string>
      </property>
      <property name="text">
       <string>Transactions</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QTableView" name="transactionstableView">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>120</y>
        <width>711</width>
        <height>431</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="budgetsPage">
     <widget class="QTableView" name="budgetstableView">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>320</y>
        <width>711</width>
        <height>251</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="addbudgetlabel">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>90</y>
        <width>711</width>
        <height>211</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel#addbudgetlabel {
    background-color: #bfa391;
	border-radius: 10px
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="budgetsLabel">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>30</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Variable Text Semibold</family>
        <pointsize>26</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel#budgetsLabel {
	color: #bfa391
}</string>
      </property>
      <property name="text">
       <string>Budgets</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="categorylabel">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>110</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Variable Text Semibold</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Category</string>
      </property>
     </widget>
     <widget class="QLabel" name="amountlabel">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>110</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Variable Text Semibold</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Amount</string>
      </property>
     </widget>
     <widget class="QComboBox" name="categorycomboBox">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>150</y>
        <width>271</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arad Medium</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
     <widget class="QLineEdit" name="amountlineEdit">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>150</y>
        <width>271</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Variable Text Semibold</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit#amountlineEdit {
    border: 2px solid #A47551;
    border-radius: 10px;
    padding: 6px;
    background-color: #bfa391;
    color: #3e2c1c;
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="savebudgetButton">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>230</y>
        <width>130</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Variable Text Semibold</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#savebudgetButton {
	background-color: #5c4033;
    color: #bfa391;
    border: none;
    border-radius: 10px;
    padding: 8px 12px
}

QPushButton#savebudgetButton:hover {
    background-color: #a37c68;
	color: black;
}</string>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="reportsPage">
     <widget class="QLabel" name="reportsLabel">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Variable Text Semibold</family>
        <pointsize>26</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel#reportsLabel {
	color: #bfa391
}</string>
      </property>
      <property name="text">
       <string>Reports</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QTableView" name="reportstableView">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>731</width>
        <height>451</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>201</width>
      <height>601</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QFrame" name="sidebar">
       <property name="styleSheet">
        <string notr="true">QFrame#sidebar {
    background-color: #5c4033;
    border: none;
}</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QPushButton" name="dashboardButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>21</y>
          <width>181</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI Variable Text Semibold</family>
          <pointsize>13</pointsize>
          <weight>75</weight>
          <bold>true</bold>
          <underline>false</underline>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton#dashboardButton {
	background-color: transparent;
    color: #f5f5dc;
    border: none;
    padding: 12px;
    text-align: left;
    border-radius: 8px;
}

QPushButton#dashboardButton:hover {
    background-color: #8d6e63;
}

QPushButton#dashboardButton:checked {
    background-color: #a1887f;
}</string>
        </property>
        <property name="text">
         <string>🏦 Dashboard</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="transactionsButton">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>70</y>
          <width>181</width>
          <height>51</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI Variable Text Semibold</family>
          <pointsize>13</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton#transactionsButton {
	background-color: transparent;
    color: #f5f5dc;
    border: none;
    padding: 12px;
    text-align: left;
    border-radius: 8px;
    font-weight: bold;
}

QPushButton#transactionsButton:hover {
    background-color: #8d6e63;
}

QPushButton#transactionsButton:checked {
    background-color: #a1887f;
}</string>
        </property>
        <property name="text">
         <string>💳 Transactions</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="budgetsButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>181</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI Variable Text Semibold</family>
          <pointsize>13</pointsize>
          <weight>75</weight>
          <bold>true</bold>
          <underline>false</underline>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton#budgetsButton {
	background-color: transparent;
    color: #f5f5dc;
    border: none;
    padding: 12px;
    text-align: left;
    border-radius: 8px;
}

QPushButton#budgetsButton:hover {
    background-color: #8d6e63;
}

QPushButton#budgetsButton:checked {
    background-color: #a1887f;
}</string>
        </property>
        <property name="text">
         <string>💰 Budgets</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="reportsButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>170</y>
          <width>181</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI Variable Text Semibold</family>
          <pointsize>13</pointsize>
          <weight>75</weight>
          <bold>true</bold>
          <underline>false</underline>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton#reportsButton {
	background-color: transparent;
    color: #f5f5dc;
    border: none;
    padding: 12px;
    text-align: left;
    border-radius: 8px;
}

QPushButton#reportsButton:hover {
    background-color: #8d6e63;
}

QPushButton#reportsButton:checked {
    background-color: #a1887f;
}</string>
        </property>
        <property name="text">
         <string>📶 Reports</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
